### ISSUE TITLE
[Bug]: Creating a new payee doesn't change focus

### ISSUE THREAD

**Issue by rich-howell** [actualbudget/actual#1014]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

When adding a new Payee from the drop down - if you select "Create Payee"

![image](https://user-images.githubusercontent.com/22135084/236895695-b07f7433-161a-47a9-967d-875320e8acca.png)

The focus doesn't change out of the Payee drop down it just stays there. 

![image](https://user-images.githubusercontent.com/22135084/236895896-eb1b25ec-0947-4c7a-bb48-3098ac36b391.png)

### What error did you receive?

None

### Where are you hosting Actual?

Fly.io

### What browsers are you seeing the problem on?

Microsoft Edge

### Operating System

Windows 11

**Comment by trevdor** [comment]
imo, both this and #941 describe `Autocomplete` component and/or transaction input workflow enhancements, not bugs. As such, we should route them through the ideas process. 

That is, the focus stays in the input when you make _any_ selection in our autocomplete dropdown component (Payee and Category being two examples). And `<tab>` is simply not a key our component was built to listen for. From what I can find, some accessibility guidance suggests supporting it, some doesn't. 

@MatissJanis did some cleanup in our `Autocomplete` component--would be curious how he's leaning today re: building or pulling an autocomplete off-the-shelf that might support these features already.

**Comment by MatissJanis** [comment]
> @MatissJanis did some cleanup in our Autocomplete component--would be curious how he's leaning today re: building or pulling an autocomplete off-the-shelf that might support these features already.

Exptending our existing autocomplete for sure. I tried porting it to an off-the-shelf solution, but there's just too many weird quirks we need to support in order not to break existing workflows. I couldn't solve them all nicely, so autocomplete-v2 was removed and I just upgraded the existing solution from Class component to a Function (+upgraded dependencies).
